### ISSUE TITLE
'ERESOLVE could not resolve' after upgrade to WDIO version 8

### ISSUE THREAD

**Issue by NaZaRKIN123** [webdriverio-community/wdio-edgedriver-service#92]
Hello All,

After I upgrade to the WDIO 8 I couldn't run NPM install.

This is the error that I get:
> npm ERR! code ERESOLVE
> npm ERR! ERESOLVE could not resolve
> npm ERR! 
> npm ERR! While resolving: wdio-edgedriver-service@2.1.1
> npm ERR! Found: @wdio/types@8.1.0
> npm ERR! node_modules/@wdio/types
> npm ERR!   dev @wdio/types@"^8.1.0" from the root project
> npm ERR!   @wdio/types@"8.1.0" from @wdio/allure-reporter@8.1.0
> npm ERR!   node_modules/@wdio/allure-reporter
> npm ERR!     dev @wdio/allure-reporter@"^8.1.0" from the root project
> npm ERR!   12 more (@wdio/cli, @wdio/cucumber-framework, ...)
> npm ERR! 
> npm ERR! Could not resolve dependency:
> npm ERR! peerOptional @wdio/types@"^7.25.0" from wdio-edgedriver-service@2.1.1
> npm ERR! node_modules/wdio-edgedriver-service
> npm ERR!   dev wdio-edgedriver-service@"^2.1.1" from the root project
> npm ERR! 
> npm ERR! Conflicting peer dependency: @wdio/types@7.26.0
> npm ERR! node_modules/@wdio/types
> npm ERR!   peerOptional @wdio/types@"^7.25.0" from wdio-edgedriver-service@2.1.1
> npm ERR!   node_modules/wdio-edgedriver-service
> npm ERR!     dev wdio-edgedriver-service@"^2.1.1" from the root project
> npm ERR! 
> npm ERR! Fix the upstream dependency conflict, or retry
> npm ERR! this command with --force, or --legacy-peer-deps
> npm ERR! to accept an incorrect (and potentially broken) dependency resolution.

This is my package.json
```
{
  "name": "test-automation-solution-webdriverio",
  "version": "1.0.0",
  "private": true,
  "devDependencies": {
    "@faker-js/faker": "^7.6.0",
    "@types/chai": "^4.3.4",
    "@types/dateformat": "^5.0.0",
    "@types/deep-equal": "^1.0.1",
    "@types/fs-extra": "^9.0.13",
    "@types/jsonpath": "^0.2.0",
    "@types/pg": "^8.6.6",
    "@types/shelljs": "^0.8.11",
    "@typescript-eslint/eslint-plugin": "^5.48.0",
    "@wdio/allure-reporter": "^8.1.0",
    "@wdio/cli": "^8.1.0",
    "@wdio/cucumber-framework": "^8.1.0",
    "@wdio/local-runner": "^8.1.0",
    "@wdio/shared-store-service": "^8.1.0",
    "@wdio/spec-reporter": "^8.1.0",
    "@wdio/types": "^8.1.0",
    "allure-js-commons": "^2.0.0-beta.23",
    "async-ray": "^3.2.0",
    "axios": "^0.27.2",
    "chai": "^4.3.7",
    "chromedriver": "^108.0.0",
    "dateformat": "^5.0.3",
    "deep-equal": "^2.1.0",
    "deepmerge": "^4.2.2",
    "eslint": "^8.31.0",
    "eslint-config-prettier": "^8.6.0",
    "eslint-plugin-check-file": "^1.3.0",
    "eslint-plugin-import": "^2.26.0",
    "eslint-plugin-prettier": "^4.2.1",
    "eslint-plugin-simple-import-sort": "^8.0.0",
    "eslint-plugin-wdio": "^8.0.14",
    "extract-zip": "^2.0.1",
    "form-data": "^4.0.0",
    "fs-extra": "^11.1.0",
    "husky": "^8.0.3",
    "jsonpath": "^1.1.1",
    "kafkajs": "^2.2.3",
    "lint-staged": "^13.1.0",
    "log4js": "^6.7.1",
    "ms-chromium-edge-driver": "^0.5.1",
    "pg": "^8.8.0",
    "prettier": "^2.8.1",
    "shelljs": "^0.8.5",
    "shx": "^0.3.4",
    "ts-node": "^10.9.1",
    "typescript": "^4.9.4",
    "wdio-chromedriver-service": "^8.0.1",
    "wdio-edgedriver-service": "^2.1.1",
    "wdio-video-reporter": "^3.4.0"
  },
  "scripts": {
    "build": "npx tsc -b tsconfig.json",
    "lint": "eslint --ext .js,.ts src config --max-warnings=0 && npm run gherkin:lint",
    "es:lint:fix": "eslint --fix --ext .js,.ts src config --max-warnings=0",
    "gherkin:lint": "npx gherkin-lint -c .gherkin-lintrc.json src/tests",
    "pre-commit": "npx prettier --write src config && npm run es:lint:fix && npm run gherkin:lint",
    "prepare": "husky install"
  },
  "lint-staged": {
    "**/*": "npm run pre-commit"
  }
}
```
My node: v18.12.1
My NPM: 8.12.0

I tried doing a clean install with node_modules and package.lock removed, and it didn't help.

**Comment by christian-bromann** [comment]
@NaZaRKIN123 thanks for raising the issue. I put in a fix for it and released `v2.1.2`. Please let me know if this resolves the problem.

**Comment by NaZaRKIN123** [comment]
yes, it helped. I'm able to run the install now. Thanks!
